docs/actions.rst

```diff
@@ -51,6 +44,9 @@ All actions in Crud, even your own, should inherit from the
 This class is ``abstract`` and provides numerous auxiliary methods which can be
 useful for you both as a developer as an action creator.
 
+Where you choose to put your custom Crud actions is up to you, but using ``src/Crud`` is a good place. Remember to
```

Comment:
Hmmm, this might be a fault on my side but after moving my custom actions to `src/Crud/Actions` the following mapping no longer works:

```php
$this->Crud->mapAction('changePassword', 'App.Crud.Actions.ChangePassword');
```
whereas the following did work before:

```php
$this->Crud->mapAction('changePassword', 'App.Actions.ChangePassword');
``` 

Stack trace:

```json
{
  "message": "Could not find action class: App.Crud.Actions.ChangePassword",
  "url": "v0/users/change-password",
  "code": 500,
  "trace": [
    {
      "file": "/home/vagrant/projects/api.app/vendor/friendsofcake/crud/src/Controller/Component/CrudComponent.php",
      "line": 262,
      "function": "_loadAction",
      "class": "Crud\\Controller\\Component\\CrudComponent",
      "type": "->",
      "args": [
        "changePassword"
      ]
    },
    {
      "file": "/home/vagrant/projects/api.app/vendor/friendsofcake/crud/src/Controller/Component/CrudComponent.php",
      "line": 274,
      "function": "action",
      "class": "Crud\\Controller\\Component\\CrudComponent",
      "type": "->",
      "args": [
        "changePassword"
      ]
    },
    {
      "file": "/home/vagrant/projects/api.app/vendor/friendsofcake/crud/src/Controller/Component/CrudComponent.php",
      "line": 374,
      "function": "enable",
      "class": "Crud\\Controller\\Component\\CrudComponent",
      "type": "->",
      "args": [
        "changePassword"
      ]
    },
    {
      "file": "/home/vagrant/projects/api.app/src/Controller/UsersController.php",
      "line": 29,
      "function": "mapAction",
      "class": "Crud\\Controller\\Component\\CrudComponent",
      "type": "->",
      "args": [
        "changePassword",
        "App.Crud.Actions.ChangePassword"
      ]
    },
    {
      "file": "/home/vagrant/projects/api.app/vendor/cakephp/cakephp/src/Controller/Controller.php",
      "line": 262,
      "function": "initialize",
      "class": "App\\Controller\\UsersController",
      "type": "->",
      "args": []
    }
}
```